### ISSUE TITLE
validator extending format can't be found

### ISSUE THREAD

**Issue by basz** [offirgolan/ember-cp-validations#386]
I have a validator that suddenly (after upgrading) can't be found and I think it is because it extends the 'format' validator.

```
vendor.js:29700 Error: Assertion Failed: Validator not found of type: identity/email-address-is-registered.
    at EmberError (vendor.js:29623)
    at assert (vendor.js:17318)
    at assert (vendor.js:29435)
    at validate (vendor.js:124596)
    at Class.validate (vendor.js:88749)
    at Class.validate (plhw-client.js:33012)
    at Class.superWrapper [as validate] (vendor.js:35345)
    at vendor.js:90144
    at vendor.js:90237
    at tryCatch (vendor.js:61494)
    at invokeCallback (vendor.js:61509)
    at publish (vendor.js:61477)
    at vendor.js:50957
    at invoke (vendor.js:11530)
    at Queue.flush (vendor.js:11594)
    at DeferredActionQueues.flush (vendor.js:11402)
    at Backburner.end (vendor.js:10716)
    at Backburner.run (vendor.js:10838)
    at vendor.js:11136
```

```
import FormatValidator from "ember-cp-validations/validators/format";

export default FormatValidator.extend({
```
Does not work

```
import BaseValidator from "ember-cp-validations/validators/base";

export default BaseValidator.extend({
```
Does work... (but obviously with missing some 'format' functionality)

Maybe because you now extends/use ember-validations (?)

### Environment

- Ember Version: 2.9.1
- Ember CLI Version: 2.9.1
- Ember CP Validations Version: 3.1.1
### Steps to Reproduce

create a custom validator that extends the 'format' validator.
and import it into a model

**Comment by offirgolan** [comment]
@basz thats pretty weird... Can you try doing a fresh install?

```
npm cache clear && bower cache clean && rm -rf node_modules bower_components dist tmp && npm i && bower i
```

**Comment by basz** [comment]
doesn't help

**Comment by offirgolan** [comment]
Can you use the `format` validator normally?

**Comment by basz** [comment]
yes I can... so must be something with my class...

but this works (so it is able to locate the class)
```
import BaseValidator from "ember-cp-validations/validators/base";
export default BaseValidator.extend({});
```

But this gives an error...
```
import FormatValidator from "ember-cp-validations/validators/format";
export default FormatValidator.extend({});
```
my model
```
import Ember from "ember";
import {validator, buildValidations} from "ember-cp-validations";

const Validations = buildValidations({
  emailAddress: {
    debounce: 280,
    validators: [
      validator('presence', {presence: true}),
      // regular format validator works
      // validator('format', {
      //   regex: /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{4,8}$/,
      //   message: 'Password must include at least one upper case letter, one lower case letter, and a number'
      // })

      // extended format validator can't be found, if I extend the format validator, but works with the base validator
      validator('identity/email-address-is-registered', {
        allowBlank: false,
        /* source http://emailregex.com */
        regex: /^[-a-z0-9~!$%^&*_=+}{\'?]+(\.[-a-z0-9~!$%^&*_=+}{\'?]+)*@([a-z0-9_][-a-z0-9_]*(\.[-a-z0-9_]+)*\.(aero|arpa|biz|com|coop|edu|gov|info|int|mil|museum|name|net|org|pro|travel|mobi|[a-z][a-z])|([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}))(:[0-9]{1,5})?$/i,
        disabled: Ember.computed.empty('model.emailAddress')
      })
    ]
  }
});

export default Ember.Object.extend(Validations, {
  emailAddress: ''
});
```

identity/email-address-is-registered is located in 'app/validators/identity/email-address-is-registered.js'

**Comment by offirgolan** [comment]
@basz not gonna lie but thats hella weird lol. Can you try to recreate this in a dummy app and see if that happens?

**Comment by basz** [comment]
https://github.com/basz/test-cp-validations

once you start typing in the field the same error occurs. When I switch 'format' for 'base' the error isn't there... happy huntin ;-)

**Comment by offirgolan** [comment]
@basz thanks for the demo app. That sped up the process a lot. Fix is now in `v3.1.2` 😸
